Resources/public/js/ezrecommendation.js

```diff
@@ -0,0 +1,77 @@
+// register HandleBars helpers when library is loaded
```

Comment:
Hmm, why would 1 file be better than 3 ? Files are compiled and packed, aren't they ?